### ISSUE TITLE
No input devices in game

### ISSUE THREAD

**Issue by datWeazel** [theNizo/linux_rocksmith#32]
* Which distro do you use?
EndeavourOS

* Which Proton or wine version do you use?
Proton Experimental

* Do you use pipewire?
Yes with wireplumber.

* What appears to be the problem? (Describe it as best as you can)
I connected my bass via the U-PHORIA UM2 to my PC. 
In game, I set the device to "direct mode" but under input devices it simply says "none" and I can't select any device.

The input device is working in pavucontrol. 

* Did you notice any other unexpected behavior?
I don't think so.

* What did you try already?
Basically tried the guide, tried disabling all other input devices in case there's something going on there. Also tried the VBASIOTest, but I can't seem to test any input devices there. Output devices work in VBASIOTest and ingame.

* Did you do any of the steps differently or leave them out?
No, followed the Arch Pipewire guide to the t.

**Comment by datWeazel** [comment]
Forgot to update RS_ASIO.
